### PR TITLE
features: Add simplerov-5

### LINKED ISSUE #5: Add simplerov-5 diagram to supported frames

### PR DESCRIPTION
**Pull request by patrickelectric**
Fix #5

Signed-off-by: Patrick José Pereira <patrickelectric@gmail.com>

### CODE REVIEW

**Comment by jaxxzer** [comment]
Can you align # 5 horizontally so it is in the middle of the frame (no yaw)?

image

**Comment by patrickelectric** [comment]
How can 5 create a yaw force ? And how is this related to this PR ?

**Comment by jaxxzer** [comment]
The way you have drawn the picture for the pull request, it looks like 5 will make the vehicle spin around (yaw) when actuated. thruster 5 should not have any yaw contribution, so you should change your picture to reflect this.

**Comment by jaxxzer** [comment]
Position your number 5 in the middle of the side thrusters like number 6 is in the brov1 picture, that I linked.

**Comment by patrickelectric** [comment]
I thought that you were talking about bluerov frame and not simplerov-5. I edited my comment after realizing that, but I think it was too late.

**Comment by jaxxzer** [comment]
I noticed, I made it confusing. sorry.

**Comment by patrickelectric** [comment]
@jaxxzer 	fixed.

File: introduction/features.md [hunk 1]
```diff
@@ -38,6 +38,10 @@ ArduSub includes a high-level motor library that can configure motors in any con
 			<img src="/images/simplerov-4.png" class="img-responsive img-center" style="max-height:250px;">
 			<p><strong>ROV</strong> with side-by-side vertical thrusters. (Frame: <code>simplerov</code>)</p>
 		</td>
+		<td align="center">
+			<img src="/images/simplerov-5.png" class="img-responsive img-center" style="max-height:250px;">
+			<p><strong>ROV</strong> with side-by-side vertical and lateral thrusters. (Frame: <code>simplerov</code>)</p>
```
Comment (jaxxzer) [inline introduction/features.md]:
```suggestion
			<p><strong>ROV</strong> with a lateral thruster and side-by-side vertical thrusters. (Frame: <code>simplerov</code>)</p>
```

**Comment by patrickelectric** [comment]
done

**Pull request merged**